airbyte-integrations/connectors/source-stripe/source_stripe/manifest.yaml

```diff
@@ -0,0 +1,232 @@
+        airbyte_secret: true
+        order: 1
+      start_date:
+        type: string
+        title: Replication start date
+        pattern: ^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$
+        description: >-
+          UTC date and time in the format 2017-01-25T00:00:00Z. Only data generated
+          after this date will be replicated.
+        default: "2017-01-25T00:00:00Z"
+        examples:
+          - "2017-01-25T00:00:00Z"
+        format: date-time
+        order: 2
+      lookback_window_days:
+        type: integer
+        title: Lookback Window in days
+        default: 0
+        minimum: 0
+        description: >-
+          When set, the connector will always re-export data from the past N days,
+          where N is the value set here. This is useful if your data is frequently updated
+          after creation. The Lookback Window only applies to streams that do not support event-based incremental syncs: Events,
+          SetupAttempts, ShippingRates, BalanceTransactions, Files, FileLinks, Refunds. More info <a
+          href="https://docs.airbyte.com/integrations/sources/stripe#requirements">here</a>
+        order: 3
+      slice_range:
+        type: integer
+        title: Data request time increment in days
+        default: 365
+        minimum: 1
+        examples: [1, 3, 10, 30, 180, 360]
+        description: >-
+          The time increment used by the connector when requesting data from the Stripe API. The bigger the value is,
+          the less requests will be made and faster the sync will be. On the other hand, the more seldom
+          the state is persisted.
+        order: 4
+      num_workers:
+        type: integer
+        title: Number of concurrent workers
+        minimum: 1
+        maximum: 20
+        default: 10
+        examples: [1, 2, 3]
+        description: >-
+          The number of worker thread to use for the sync.
+          The performance upper boundary depends on call_rate_limit setting and type of account.
+        order: 5
+      call_rate_limit:
+        type: integer
+        title: Max number of API calls per second
+        examples: [25, 100]
+        description: >-
+          The number of API calls per second that you allow connector to make. This value can not be bigger than real
+          API call rate limit (https://stripe.com/docs/rate-limits). If not specified the default maximum is 25 and 100
+          calls per second for test and production tokens respectively.
+
+api_budget:
+  type: HTTPAPIBudget
+  policies:
+    - type: MovingWindowCallRatePolicy
+      rates:
+        - limit: 20
+          interval: PT1S
+      matchers:
+        - method: GET
+          url_path_pattern: "^/v1/files($|/)" # matches '/files'
+        - method: GET
+          url_path_pattern: "^/v1/file_links($|/)" # matches '/file_links'
+
+    - type: MovingWindowCallRatePolicy
+      rates:
+        - limit: 25
```

Comment:
Do we put `25` here because it is the lowest limit between the test account (25 requests/sec) and the prod account (100 requests/sec) and we don't have a way to make this configurable today?

Reply:
Updated interpolation string